Makefile

```diff
@@ -165,36 +168,67 @@ endif
 
 .PHONY: bump-extension-version
 bump-extension-version:  ## (vscode) Bumps extension version
+ifeq ($(RELEASE),"true")
 	go run ci/bump_extension_version.go $(VERSION)
 	touch $@
+endif
+
+#----------------------------------------------------------------------------------
+# Deployment Manifests / Helm
+#----------------------------------------------------------------------------------
+
+HELM_SYNC_DIR := $(OUTPUT_DIR)/helm
+HELM_DIR := install/helm/$(SOLO_NAME)
+INSTALL_NAMESPACE ?= $(SOLO_NAME)
 
+.PHONY: manifest
+manifest: prepare-helm install/squash.yaml update-helm-chart
+
+# creates Chart.yaml, values.yaml
+.PHONY: prepare-helm
+prepare-helm:
+	go run install/helm/squash/generate/cmd/generate.go $(IMAGE_TAG) $(CONTAINER_REPO_ORG)
+
+.PHONY: update-helm-chart
+update-helm-chart:
+	mkdir -p $(HELM_SYNC_DIR)/charts
+	helm package --destination $(HELM_SYNC_DIR)/charts $(HELM_DIR)
+	helm repo index $(HELM_SYNC_DIR)
+
+HELMFLAGS := --namespace $(INSTALL_NAMESPACE) --set namespace.create=true
+
+install/$(SOLO_NAME).yaml: prepare-helm
+	helm template install/helm/squash $(HELMFLAGS) > $@
+
+.PHONY: render-yaml
+render-yaml: install/squash.yaml
 
 #----------------------------------------------------------------------------------
 # Build All
 #----------------------------------------------------------------------------------
 .PHONY: build
 build: squashctl squash plank
 
-
 #----------------------------------------------------------------------------------
 # Docker
 #----------------------------------------------------------------------------------
 .PHONY: docker
 docker: $(OUTPUT_DIR)/plank-dlv-container $(OUTPUT_DIR)/plank-gdb-container $(OUTPUT_DIR)/squash-container
 
-DOCKER_IMAGES :=
-ifeq ($(RELEASE),"true")
-	DOCKER_IMAGES := docker
-endif
-
 .PHONY: docker-push
-docker-push: $(DOCKER_IMAGES)
-ifeq ($(RELEASE),"true")
-	docker push $(DOCKER_REPO)/plank-dlv:$(VERSION) && \
-	docker push $(DOCKER_REPO)/plank-gdb:$(VERSION) && \
-	docker push $(DOCKER_REPO)/squash:$(VERSION)
-endif
+docker-push: docker
+	docker push $(CONTAINER_REPO_ORG)/plank-dlv:$(IMAGE_TAG) && \
+	docker push $(CONTAINER_REPO_ORG)/plank-gdb:$(IMAGE_TAG) && \
+	docker push $(CONTAINER_REPO_ORG)/squash:$(IMAGE_TAG)
+
+# TODO(mitchdraft) - replace this with a go lib when it's available
+$(OUTPUT_DIR)/buildtimevalues.yaml:
+	echo plank_image_tag: $(IMAGE_TAG) > $@
```

Comment:
if we move this to a golib we can import these strings and reduce the risk of drift/bugs

Reply:
as discussed over slack, we should move away from this towards build id or commit / describe based